libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -95,6 +95,7 @@ singleton AP_BattMonitor method get_cell_voltage boolean uint8_t'skip_check uint
 include AP_GPS/AP_GPS.h
 
 singleton AP_GPS depends (AP_GPS_ENABLED && (!defined(HAL_BUILD_AP_PERIPH) || defined(HAL_PERIPH_ENABLE_GPS)))
+singleton AP_GPS semaphore
```

Comment:
Adding the semaphore at this point mean that all the methods call it. Why not just add it in the cpp for the `handle_external` function? Like wise for all the others.